obs-studio-server/source/main.cpp

```diff
@@ -192,18 +211,13 @@ int main(int argc, char* argv[])
 
 	// Initialize Server
 	try {
-		std::string socketPath = "";
-#ifdef __APPLE__
-		socketPath = "/tmp/";
-#endif
-		socketPath += argv[1];
 		myServer.initialize(socketPath.c_str());
 	} catch (std::exception& e) {
 		std::cerr << "Initialization failed with error " << e.what() << "." << std::endl;
-		return -2;
+		return 253;
```

Comment:
See above, I wouldn't put these like this but I would put them in constant variables in lib-streamlabs-ipc